docs/en/sql-reference/statements/update.md

```diff
@@ -0,0 +1,115 @@
+It is called "lightweight update" to contrast it to the [ALTER TABLE ... UPDATE](/sql-reference/statements/alter/update) query, which is a heavyweight process that rewrites entire columns in data parts.
+It is only available for the [MergeTree](/engines/table-engines/mergetree-family/mergetree) table engine family.
+
+```sql
+UPDATE [db.]table SET column1 = expr1 [, ...] [ON CLUSTER cluster] [IN PARTITION partition_expr] WHERE filter_expr;
+```
+
+The `filter_expr` must be of type `UInt8`. This query updates values of specified columns to the values of corresponding expressions in rows for which the `filter_expr` takes a non-zero value.
+Values are cast to the column type using the `CAST` operator. Updating columns that are used in the calculation of the primary or the partition key is not supported.
+
+## Examples {#examples}
+
+```sql
+UPDATE hits SET Title = 'Updated Title' WHERE EventDate = today();
+
+UPDATE wikistat SET hits = hits + 1, time = now() WHERE path = 'ClickHouse';
+```
+
+## Lightweight updates do not update data immediately {#lightweight-update-does-not-update-data-immediately}
+
+Lightweight `UPDATE` is implemented using **patch parts** - a special type of data parts that contain only the updated columns and rows.
+Lightweight `UPDATE` creates patch parts but does not immediately modify the original data physically in storage.
+The process of update is similar to `INSERT ... SELECT ...` query. `UPDATE` query waits until the patch part creation is completed before returning.
+
+The updated values are:
+   - **Immediately visible** in `SELECT` queries through patches application
+   - **Physically materialized** only during subsequent merges and mutations
+   - **Automatically cleaned up** once all active parts have the patches materialized
+
+## Lightweight updates requirements {#lightweight-update-requirements}
+
+Lightweight updates are supported for [MergeTree](/engines/table-engines/mergetree-family/mergetree), [ReplacingMergeTree](/engines/table-engines/mergetree-family/replacingmergetree), [CollapsingMergeTree](/engines/table-engines/mergetree-family/collapsingmergetree) engines and their [Replicated](/engines/table-engines/mergetree-family/replication.md) and [Shared](/cloud/reference/shared-merge-tree) versions.
+
+To use lightweight updates, materialization of `_block_number` and `_block_offset` columns must be enabled by table settings [enable_block_number_column](/operations/settings/merge-tree-settings#enable_block_number_column) and [enable_block_offset_column](/operations/settings/merge-tree-settings#enable_block_offset_column).
+
+## Lightweight deletes {#lightweight-delete}
+
+[Lightweight DELETE](/sql-reference/statements/delete) query can be run as a lightweight `UPDATE` instead of `ALTER UPDATE` mutation. The implementation of lightweight `DELETE` is controlled by setting [lightweight_delete_mode](/operations/settings/settings#lightweight_delete_mode).
```

Comment:
```suggestion
A [lightweight `DELETE`](/sql-reference/statements/delete) query can be run as a lightweight `UPDATE` instead of a `ALTER UPDATE` mutation. The implementation of lightweight `DELETE` is controlled by setting [`lightweight_delete_mode`](/operations/settings/settings#lightweight_delete_mode).
```